src/ng/compile.js

```diff
@@ -186,6 +186,27 @@
  *   you want to shallow watch for changes (i.e. $watchCollection instead of $watch) you can use
  *   `=*` or `=*attr` (`=*?` or `=*?attr` if the property is optional).
  *
+ * * `<` or `<attr` - set up one-way (one-directional) binding between a local scope property and the
```

Comment:
How about `<=`? It has the same connatation of moving from attribute to property (even more so) it has an element of `=` about it and looks nice `propertyName: '<=attrName'`